dev/com.ibm.ws.webcontainer/src/com/ibm/wsspi/webcontainer/WCCustomProperties.java

```diff
@@ -316,6 +316,9 @@
     //20.0.0.8
     public static boolean DECODE_URL_PLUS_SIGN;
 
+    //20.0.0.12
+    public static boolean REDIRECT_TO_RELATIVE_URL;
```

Comment:
Phu I think we need to update this comment to reflect whatever release this is going to make it into. Please check it out, update the comment, run a build and lets get this merged this week if we can.